cmd/image-inspector.go

```diff
@@ -15,7 +15,8 @@ import (
 func main() {
 	inspectorOptions := iicmd.NewDefaultImageInspectorOptions()
 
-	flag.StringVar(&inspectorOptions.URI, "docker", inspectorOptions.URI, "Daemon socket to connect to")
+	flag.BoolVar(&inspectorOptions.UseDockerSocket, "use-docker-socket", inspectorOptions.UseDockerSocket, "Use docker daemon to handle image pulls (requires docker-socket)")
```

Comment:
do we need this parameter or can we imply that if `docker-socket` is set then we are using it and if not we aren't?